stravalib/client.py

```diff
@@ -686,13 +683,24 @@ def upload_activity(self, activity_file, data_type, name=None, description=None,
                               nordicski, alpineski, backcountryski, iceskate, inlineskate,
                               kitesurf, rollerski, windsurf, workout, snowboard, snowshoe
                               Type detected from file overrides, uses athlete's default type if not specified
+                              WARNING - This param is supported (as of 2022-11-15), but not
```

Comment:
@jsamoocha so is this not documented in the strava API docs?? or in our docs?  many thanks

Reply:
`activity_type` doesn't officially exist anymore as a request parameter per the Strava API docs. However, it still seems to be functioning - I manually checked that providing this parameter makes strava assign the type to the uploaded activity.